java/code/src/com/redhat/rhn/manager/audit/ScapManager.java

```diff
@@ -463,7 +465,10 @@ public static XccdfTestResult xccdfEval(Server server, ScapAction action,
         StreamSource in = new StreamSource(resultsXml);
         try (OutputStream resumeOut = new FileOutputStream(output)) {
             StreamResult out = new StreamResult(resumeOut);
-            TransformerFactory factory = TransformerFactory.newInstance();
+            SaxonTransformerFactory factory = new SaxonTransformerFactory();
```

Comment:
Why is it still needed to instantiate the `SaxonTransformerFactory` directly? Maybe we should add a comment to explain the reason, so future refactoring won't alter that.

Reply:
I couldn't get it to be created using the `TransformerFactory`, but may be I miss something.